<?xml version="1.0"?>
<!--  ==========================================================================================  -->
<!--  |    This document was autogenerated by scene_to_spawner from pirov.xacro                |  -->
<!--  |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                        |  -->
<!--  ==========================================================================================  -->
<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="pirov">
  <link name="base_link">
    <visual>
      <origin rpy="1.57079632679 0 1.57079632679" xyz="-0.253 -0.167 -0.19"/>
      <!--origin xyz="-0.241 0.165 0.17" rpy="${PI/2} 0 -${PI/2}"/-->
      <geometry>
        <mesh filename="mesh/brov-redux.osg" scale="0.001 0.001 0.001"/>
      </geometry>
    </visual>
    </link>
  <link name="body">
    </link>
  <joint name="body_offset" type="fixed">
    <parent link="base_link"/>
    <child link="body"/>
    <origin xyz="0.02 0 0"/>
  </joint>
  <link name="camera">
    <origin rpy="0 1.57079632679 0"/>
    <visual>
      <geometry>
        <cylinder length="0.03" radius="0.01"/>
      </geometry>
    </visual>
    </link>
  <joint name="tilt" type="revolute">
    <parent link="body"/>
    <child link="camera"/>
    <origin xyz="0.15 0 0"/>
    <axis xyz="0 1 0"/>
    <limit effort="100" lower="-0.785398163397" upper="0.785398163397" velocity="5.0"/>
  </joint>
  <link name="lsm"/>
  <link name="mpu"/>
  <joint name="lsm" type="fixed">
    <parent link="body"/>
    <child link="lsm"/>
    <origin rpy="0 0 0" xyz="0.1 0.018 0.02"/>
  </joint>
  <joint name="mpu" type="fixed">
    <parent link="body"/>
    <child link="mpu"/>
    <origin rpy="0 0 0" xyz="0.11 0.018 0.02"/>
  </joint>
  <joint name="thr1" type="fixed">
    <parent link="base_link"/>
    <child link="thr1"/>
    <origin rpy="-0.0 -1.57079632679 0.0" xyz="0.0 -0.109 -0.02"/>
  </joint>
  <link name="thr1">
    <!--visual name="${name}">
                <geometry>
                    <cylinder length=".5" radius=".01"/>
                </geometry>
                <material name="black">
                    <color rgba="0 0 0 1"/>
                </material>
            </visual>  
            <inertial>
                <mass value=".1"/>
                <inertia
                    ixx=".1"
                    ixy="0.0"
                    ixz="0.0"
                    iyy=".1"
                    iyz="0.0"
                    izz=".1"/>
                </inertial-->
  </link>
  <joint name="thr2" type="fixed">
    <parent link="base_link"/>
    <child link="thr2"/>
    <origin rpy="-0.0 -1.57079632679 0.0" xyz="0.0 0.109 -0.02"/>
  </joint>
  <link name="thr2">
    <!--visual name="${name}">
                <geometry>
                    <cylinder length=".5" radius=".01"/>
                </geometry>
                <material name="black">
                    <color rgba="0 0 0 1"/>
                </material>
            </visual>  
            <inertial>
                <mass value=".1"/>
                <inertia
                    ixx=".1"
                    ixy="0.0"
                    ixz="0.0"
                    iyy=".1"
                    iyz="0.0"
                    izz=".1"/>
                </inertial-->
  </link>
  <joint name="thr3" type="fixed">
    <parent link="base_link"/>
    <child link="thr3"/>
    <origin rpy="0.0 -0.0 0.0" xyz="0.165 -0.11 -0.02"/>
  </joint>
  <link name="thr3">
    <!--visual name="${name}">
                <geometry>
                    <cylinder length=".5" radius=".01"/>
                </geometry>
                <material name="black">
                    <color rgba="0 0 0 1"/>
                </material>
            </visual>  
            <inertial>
                <mass value=".1"/>
                <inertia
                    ixx=".1"
                    ixy="0.0"
                    ixz="0.0"
                    iyy=".1"
                    iyz="0.0"
                    izz=".1"/>
                </inertial-->
  </link>
  <joint name="thr4" type="fixed">
    <parent link="base_link"/>
    <child link="thr4"/>
    <origin rpy="0.0 -0.0 0.0" xyz="0.165 0.11 -0.02"/>
  </joint>
  <link name="thr4">
    <!--visual name="${name}">
                <geometry>
                    <cylinder length=".5" radius=".01"/>
                </geometry>
                <material name="black">
                    <color rgba="0 0 0 1"/>
                </material>
            </visual>  
            <inertial>
                <mass value=".1"/>
                <inertia
                    ixx=".1"
                    ixy="0.0"
                    ixz="0.0"
                    iyy=".1"
                    iyz="0.0"
                    izz=".1"/>
                </inertial-->
  </link>
  <joint name="thr5" type="fixed">
    <parent link="base_link"/>
    <child link="thr5"/>
    <origin rpy="0.0 -0.0 0.0" xyz="-0.185 0.0 -0.105"/>
  </joint>
  <link name="thr5">
    <!--visual name="${name}">
                <geometry>
                    <cylinder length=".5" radius=".01"/>
                </geometry>
                <material name="black">
                    <color rgba="0 0 0 1"/>
                </material>
            </visual>  
            <inertial>
                <mass value=".1"/>
                <inertia
                    ixx=".1"
                    ixy="0.0"
                    ixz="0.0"
                    iyy=".1"
                    iyz="0.0"
                    izz=".1"/>
                </inertial-->
  </link>
  <joint name="thr6" type="fixed">
    <parent link="base_link"/>
    <child link="thr6"/>
    <origin rpy="1.57079632679 -0.0 0.0" xyz="0.028 0.0 -0.127"/>
  </joint>
  <link name="thr6">
    <!--visual name="${name}">
                <geometry>
                    <cylinder length=".5" radius=".01"/>
                </geometry>
                <material name="black">
                    <color rgba="0 0 0 1"/>
                </material>
            </visual>  
            <inertial>
                <mass value=".1"/>
                <inertia
                    ixx=".1"
                    ixy="0.0"
                    ixz="0.0"
                    iyy=".1"
                    iyz="0.0"
                    izz=".1"/>
                </inertial-->
  </link>
  </robot>
